services/web/client/source/class/osparc/component/widget/NodePorts.js

```diff
@@ -44,7 +44,20 @@ qx.Class.define("osparc.component.widget.NodePorts", {
 
     this.base(arguments, node.getLabel());
 
+    this.getTitleBar().set({
+      height: 30
+    });
     node.bind("label", this, "title");
+
+    node.getStatus().bind("modified", this.getChildControl("icon"), "source", {
+      converter: modified => osparc.utils.StatusUI.getIconSource(modified === true ? "modified" : "up-to-date")
+    }, this);
+    node.getStatus().bind("modified", this.getChildControl("icon"), "textColor", {
+      converter: modified => osparc.utils.StatusUI.getColor(modified === true ? "modified" : "up-to-date")
+    }, this);
+    node.getStatus().bind("modified", this.getChildControl("icon"), "toolTipText", {
+      converter: modified => modified === true ? this.tr("Out of date") : ""
```

Comment:
In the outputs, I would replace "Out of date" by "Pending" or "Still not ready" or "Soon in the best theaters"? @KZzizzle any suggestion?

Reply:
Not really. It doesn't mean "pending" or "still not ready". The outputs can be there but not aligned with the current inputs/settings. It might get a bit more clear when I implement Katie's request. No problem changing the "Out of date" text to something else though.